src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -214,7 +212,6 @@ const mapStateToProps = state => ({
   isStartingOver: state.form?.isStartingOver,
   loggedIn: isLoggedIn(state),
   mvi: state.mvi,
-  pdfLimit: uploadPdfLimitFeature(state),
```

Comment:
Please remove:

- the code and comments around line 196

    ```js
    sessionStorage.setItem(PDF_SIZE_FEATURE, pdfLimit);
    ```
- `PDF_SIZE_FEATURE` on line 26 and in the constants.js file
- `pdfLimit` on line 74

Reply:
@Mottie Do you think anything should be here? https://github.com/department-of-veterans-affairs/vets-website/blob/38842-remove-evss_upload_limit_150mb-flag/src/applications/disability-benefits/all-claims/utils/schemas.js#L256

Reply:
That can be removed too.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/c2eaeb23eaadc74b63a7379009b56a9f459d3822/src/applications/disability-benefits/all-claims/utils/schemas.js#L289. This line can be changed to:
```js
maxPdfSize: MAX_PDF_FILE_SIZE_BYTES,
```